ci/taos/plugins-base/pr-postbuild-build-android.sh

```diff
@@ -213,7 +213,7 @@ function pr-postbuild-build-android-run-queue(){
 
         # Android SDK
         android_sdk=/var/www/ubuntu/Android/Sdk
-        android_ndk=$android_sdk/ndk-bundle
+        android_ndk=$ROOT_ANDROID_CI/android-ndk-r20b
```

Comment:
Do we have to set-up $ROOT_ANDROID_CI/ folder manually?

Reply:
Let's keep the existing variable. 
export ROOT_ANDROID_CI=/var/www/html/android